taxi/urls.py

```diff
@@ -39,13 +43,34 @@
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path(
+        "cars/<int:pk>/assign/",
+        views.assign_driver,
+        name="car-assign-driver"
+    ),
+    path(
+        "cars/<int:pk>/remove/",
+        views.remove_driver,
+        name="car-remove-driver"
```

Comment:
The URL name 'car-remove-driver' is inconsistent with the naming convention used elsewhere in the file. Consider using 'cars-remove-driver' for consistency.